content/learn/book/welcome/hello-bevy/_index.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "Hello, Bevy!"
+weight = 3
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+So, you've just made a project with Bevy. Great! However, currently, it isn't doing things 'the Bevy way'. Let's fix.
+
+## Bevy App Structure
+
+Bevy programs, which include games, tools and more, are conventially named 'Apps' and currently our Bevy program doesn't really follow the app structure or how bevy apps work.
+
+Below is a basic outline of how your Bevy Apps should look.
```

Comment:
This is an interesting idea to show the structural elements and code organization. Hmm....